chromium_src/tools/grit/preprocess_if_expr.py

```diff
@@ -44,15 +44,44 @@ def maybe_keep_upstream_version(override_in_folder, out_folder, override_file):
     return None
 
 
+def run_mangler(mangler_file, preprocess_file):
+    """Runs the mangler on the given file"""
+    import subprocess
+    import brave_node
+
+    tsx = brave_node.PathInNodeModules('tsx', 'dist', 'cli.mjs')
+    ts_config = os.path.join(get_src_dir(), 'brave', 'tsconfig-mangle.json')
```

Comment:
nit: `wspath('//brave/tsconfig-mangle.json')`. same for the path below.

Reply:
Great tip :+1: